### ISSUE TITLE
Object renaming bug

### ISSUE THREAD

**Issue by SomeGuy55** [YellowAfterlife/GMEdit#177]
This happens with both the latest stable and beta releases.

In GMS 1.4 projects, when an object is renamed in GMEdit, a suffix `.object.gmx` is erroneously added to the object's name in the project file. But in a GMS 1.4 project file, objects should be only referenced by their names without any extension.
Instead of `<object>objects\obj_cube.object.gmx</object>`, it should be just `<object>objects\obj_cube</object>`

The error described makes an object seem corrupted in both GMEdit and GM's IDE. But using a text editor to manually remove the suffix after the object's name in the project file, resolves that issue.

In short, GMEdit should simply avoid adding the `.object.gmx` suffix to object names in GMS 1.4 project files.
